src/alchemlyb/tests/parsing/test_amber.py

```diff
@@ -11,14 +12,66 @@
 from alchemtest.amber import load_invalidfiles
 from alchemtest.amber import load_bace_example
 from alchemtest.amber import load_bace_improper
+from alchemtest.amber import load_single_u_nk_file
+from alchemtest.amber import load_single_dHdl_file
 
 
-@pytest.fixture(scope="module",
-                params=[filename for filename in load_invalidfiles()['data'][0]])
-def invalid_file(request):
+@pytest.fixture(
+    name="invalid_file", scope="module",
+    params=list(load_invalidfiles()['data'][0]))
+def fixture_invalid_file(request):
+    """
+    Set 'invalid_file' for the subsequent tests, 
+    returning each file in load_invalidfiles
+    """
     return request.param
 
 
+@pytest.fixture(name="single_u_nk", scope="module")
+def fixture_single_u_nk():
+    """return a single file to check u_unk parsing"""
+    return load_single_u_nk_file()[0]
+
+
+@pytest.fixture(name="single_dHdl", scope="module")
+def fixture_single_dHdl():
+    """return a single file to check u_unk parsing"""
+    return load_single_dHdl_file()[0]
```

Comment:
see https://github.com/alchemistry/alchemtest/pull/66#discussion_r969934092 for how to get a single file from `load_simple_solvated()`